decide/booth/static/js/grafica_visualizer.js

```diff
@@ -0,0 +1,124 @@
+var ctx1 = document.getElementById('bar-resultados').getContext('2d');
+var ctx2 = document.getElementById('pie-porcentaje').getContext('2d');
+
+var votacion = JSON.parse(document.getElementById('votacion').value);
+var lista = votacion.postproc;
+var votos = [];
+var etiquetas = [];
+var porVotosPorOpcion = [];
+var totalVotos = 0;
+
+for(i=0; i< lista.length ; i++){
+    votos.push(lista[i].votes);
+    etiquetas.push(lista[i].option);
+    totalVotos += lista[i].votes;
+}
+
+for(i=0; i< votos.length ; i++){
+    if(votos[i] != 0 && totalVotos != 0){
+        porVotosPorOpcion.push((votos[i]/totalVotos)*100);
+    } else {
+        porVotosPorOpcion.push(votos[i]);
+    }
+}
+
+var config1 = new Chart(ctx1, {
+    type: 'bar',
+    data: {
+        datasets: [{
+            label: 'Resultados de la votación',
+            data: votos,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.4)',
+                'rgba(255, 159, 64, 0.4)',
+                'rgba(255, 205, 86, 0.4)',
+                'rgba(75, 192, 192, 0.4)',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/egc-veleta-g1/decide/pullRequest?prid=8602013)
```suggestion
                "rgba(75, 192, 192, 0.4)",
```